src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,129 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final float CAPACITY_GROWTH = 1.5f;
+    private static final ArrayListIndexOutOfBoundsException
+            ARRAY_LIST_INDEX_OUT_OF_BOUNDS_EXCEPTION
+            = new ArrayListIndexOutOfBoundsException(
+                    "Given index is not valid for size");
+    private T[] elementData = (T[]) new Object[DEFAULT_CAPACITY];
```

Comment:
Let's better initialize it within constructor. Also I'd recommend renaming it to 'values'